pkg/util/winutil/pdhutil/pdhhelper.go

```diff
@@ -0,0 +1,176 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+// +build windows
+
+package pdhutil
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+	"github.com/lxn/win"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	modPdhDll = syscall.NewLazyDLL("pdh.dll")
+
+	procPdhLookupPerfNameByIndex    = modPdhDll.NewProc("PdhLookupPerfNameByIndexW")
+	procPdhEnumObjectItems          = modPdhDll.NewProc("PdhEnumObjectItemsW")
+	procPdhMakeCounterPath          = modPdhDll.NewProc("PdhMakeCounterPathW")
+	procPdhGetFormattedCounterValue = modPdhDll.NewProc("PdhGetFormattedCounterValue")
+)
+
+const (
+	// taken from winperf.h
+	PERF_DETAIL_NOVICE   = 100 // The uninformed can understand it
+	PERF_DETAIL_ADVANCED = 200 // For the advanced user
+	PERF_DETAIL_EXPERT   = 300 // For the expert user
```

Comment:
Are these unused? Can they be removed? if there's a good reason for keeping them here it's okay.

Reply:
Again, more for completeness than anything else.  We're providing access to the full API, so I thought we should have all the flags.